tests/data/repos/package_1/tool_dependencies.xml

```diff
@@ -4,35 +4,35 @@
     <install version="1.0">
       <actions_group>
         <actions os="linux" architecture="i386">
-          <action type="download_by_url" target_filename="samtools-0.1.16.tgz">http://depot.galaxyproject.org/package/linux/i386/samtools/samtools-0.1.16-linux-i386.tgz</action>
+          <action type="download_by_url" target_filename="samtools-0.1.16.tgz" sha256="7090bd62142df0ed3a2b4000292fc0edc917e773d0269c2f3f5b0d36b7888c22">https://depot.galaxyproject.org/package/linux/i386/samtools/samtools-0.1.16-linux-i386.tgz</action>
```

Comment:
```sha256=``` = ```sha256sum``` (if sha256 is desired, please change it in *lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py*)